pod-e2e.sh

```diff
@@ -10,7 +10,7 @@ POD_NAME="pod_e2e"
 
 echo "cleaning up and recreating pod"
 podman pod rm -f ${POD_NAME}
-podman pod create --name=${POD_NAME} --publish 8000
+podman pod create --name=${POD_NAME} --publish 8000:8000
```

Comment:
Using the isolated pod structure I created (and assuming you also put your Cypress tests inside a container in the pod), there's no need to expose any ports at all: the pod can be & should be completely isolated.

Reply:
the cypress tests can be done in a container, but they don't need to be. I left it this way to be able to reuse this script to setup the dependencies for local development of the e2e tests with the cypress application. I haven't figured out how to make that work when the frontend is containerized.